definitions/divorce/json/CaseField/CaseField.json

```diff
@@ -5284,5 +5284,48 @@
     "Label": "This event returns the case to the state it was in before the general referral was made e.g. if the case was in ‘Awaiting DN application’ when the general referral was created, then it will be returned to ‘Awaiting DN application’.",
     "FieldType": "Label",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "11/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "JudgeCostsDecision",
```

Comment:
I know @yarpo prefers that if a toggle is required, we will need to move all fields and auths into a -nonprod file